documentation/partner_backend.md

```diff
@@ -300,19 +319,27 @@ The Refresh Token ensures the user does not have to be issued a Verification rep
 	* `scope`: List of scopes included in the token (same as the value provided in the HTTP request.
 	* `id_token`: Unused field.
 
-	In case of an error, the API will return an HTTP 500 and a status in the "error_description" field:
+	In case of an error, the API will return the following error codes:
 
-	* `BAD_REQUEST`: Missing "referesh_token" parameter.
+	* HTTP 401
+		* `error="access_denied"`: missing or invalid credentials.
 
-		⇒ Check the API request to make sure all fields have been populated.
+			⇒ Check the request is using the correct value for `client_id` and `client_secret`.
 
-	* `UNAUTHORIZED_API`: Missing or invalid "audience" parameter.
+	* HTTP 403
+		* `"error": "invalid_grant"`: Missing or invalid grant parameters.
 
-		⇒ Check the API request to make sure the "audience" field has the right value.
+			⇒ Check the API request to make sure the "refresh_token" field has the right value.
+
+		* `"error": "unauthorized_client"`: Grant type not allowed for the client.
+
+			⇒ Check the API request to make sure the "grant_type" field has the right value.
+
+	* HTTP 500
+		* `error="internal_server_error"`: there was an unexpected error.
 
-	* `INTERNAL_SERVER_ERROR`: There was an unexpected error.
+			⇒ Contact Latch Support to help debug this issue.
 
-		⇒ Contact Latch Support to help debug this issue.
```

Comment:
Similar to above comment, this API returns 400 error code if request payload is missing:
1. `grant_type`
2. `refresh_token`

Lets add those scenarios as well.

Reply:
Done!